### PR TITLE
Update text

### PR DESCRIPTION
**Pull request by zishanmirza**
This patch updates switches "Owner" with "Researcher" and updates the text.

- [x] Tests passing
- [X] Black formatting
- [ ] Migrations for any changes to the database schema
- [X] Rebase/merge the `dev` branch
- [ ] Note in the CHANGELOG

### CODE REVIEW

**Comment by zishanmirza** [comment]
Fixes https://github.com/ScilifelabDataCentre/dds_cli/issues/345.

File: dds_web/api/user.py [hunk 1]
```diff
@@ -383,14 +383,14 @@ def add_to_project(whom, project, role, send_email=True):
             AddUser.compose_and_send_email_to_user(whom, "project_release", project=project)
 
         flask.current_app.logger.debug(
-            f"{str(whom)} was associated with {str(project)} as Owner={is_owner}."
+            f"{str(whom)} was associated to the {str(project)} as a Researcher={is_owner}."
```
Comment (inaod568) [inline dds_web/api/user.py:386]: Ah, actually what I meant is something like this: (removing the {is_owner} part)

```suggestion
            f"{str(whom)} was associated to the {str(project)} as a {'Project Owner' if is_owner else 'Researcher'}."
```

File: dds_web/api/user.py [hunk 2]
```diff
@@ -383,14 +383,14 @@ def add_to_project(whom, project, role, send_email=True):
...
         )
 
         return {
             "status": http.HTTPStatus.OK,
             "message": (
-                f"{str(whom)} was associated with "
-                f"{str(project)} as Owner={is_owner}. An e-mail notification has{' not ' if not send_email else ' '}been sent."
+                f"{str(whom)} was associated to the "
+                f"{str(project)} as a Researcher={is_owner}. An e-mail notification has{' not ' if not send_email else ' '}been sent."
```
Comment (inaod568) [inline dds_web/api/user.py:393]: Same here
```suggestion
                f"{str(project)} as a {'Project Owner' if is_owner else 'Researcher'}. An e-mail notification has{' not ' if not send_email else ' '}been sent."
```

File: tests/test_project_status.py [hunk 3]
```diff
@@ -379,7 +379,7 @@ def test_set_project_to_available_no_mail(module_client, boto3_session):
     assert response.status_code == http.HTTPStatus.OK
     assert response.json and response.json.get("user_addition_statuses")
     for x in response.json.get("user_addition_statuses"):
-        assert "associated with Project" in x
+        assert "associated to the Project" in x
```
Comment (inaod568) [inline tests/test_project_status.py:382]:
```suggestion
        assert "given access to the Project" in x
```

**Review by inaod568 — CHANGES_REQUESTED** [review]
A few changes to remove the "={}" part

**Review by inaod568 — APPROVED** [review]

**Comment by inaod568** [comment]
Looks good!

**Pull request merged**